server/src/controllers/loans.controllers.ts

```diff
@@ -0,0 +1,40 @@
+import OutstandingLoan from "../models/OutstandingLoan";
+import {type Request, type Response} from "express";
+
+const { ObjectId } = require("mongoose").Types;
+
+const editLoan = async (req: Request, res: Response): Promise<any> => {
+  try {
+    const id = new ObjectId(req.body._id);
+    if (id) {
+      console.log(req.body);
+      await OutstandingLoan.findByIdAndUpdate(id, {$set: req.body.content})
+        .then((result) => res.status(200).json(result))
```

Comment:
.catch here is unnecessary, see example here for a cleaner way to do this: https://github.com/ChangePlusPlusVandy/vac-net/blob/a9c4e9bab9f8ff75490e1b4e561bcb6290a39fbc/server/src/controllers/session.controllors.ts#L9